src/main/kotlin/com/kotlin/boardproject/global/util/UserPrincipal.kt

```diff
@@ -1,15 +1,16 @@
-package com.kotlin.boardproject.auth
+package com.kotlin.boardproject.global.util
 
-import com.kotlin.boardproject.common.enums.Role
-import com.kotlin.boardproject.model.User
+import com.kotlin.boardproject.domain.user.domain.User
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end